bin/latest-stable

```diff
@@ -11,4 +11,4 @@ get_maven_versions() {
   done
 }
 
-get_maven_versions | sort -r | head -n 1
+get_maven_versions | sort -t. -k1,1nr -k2,2nr -k3,3nr | head -n 1
```

Comment:
can be simplified with `-V/--version-sort`. Also sorting in `list-all` needs to use the same sorting

Reply:
@artemy - for asdf this must not be used since it is not supported in all cases.

Reply:
good to know